workflow-service/src/test/java/com/redhat/parodos/workflow/execution/controller/WorkFlowControllerTest.java

```diff
@@ -102,6 +106,68 @@ public void TestGetStatusWithoutAuth() throws Exception {
 				.andExpect(MockMvcResultMatchers.status().isUnauthorized());
 	}
 
+	@Test
+	public void updateWorkFlowCheckerTaskStatusWithValidData() throws Exception {
+		// given
+		UUID workFlowExecutionId = UUID.randomUUID();
+		String workFlowCheckerTaskName = "testWorkflowCheckerTaskName";
+
+		// when
+		doNothing().when(this.workFlowService).updateWorkFlowCheckerTaskStatus(Mockito.any(), Mockito.any(),
+				Mockito.any());
+
+		// then
+		this.mockMvc
+				.perform(this
+						.postRequestWithValidCredentials(String.format("/api/v1/workflows/%s/checkers/%s",
+								workFlowExecutionId, workFlowCheckerTaskName))
+						.content(getWorkFlowCheckerTaskRequestDTOJsonPayload()))
```

Comment:
Do you think it's possible to improve the formatting? I think It may be a bit difficult to read.

Reply:
Yes! It has been updated.